app/_src/guides/federate.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Federate zone control plane
+---
+
+With {{site.mesh_product_name}} you can first start with just one zone control plane
+and then federate it to a multi-zone deployment.
+This way you can:
+- see your mesh deployment in one centralized place
+- connect multiple zones and introduce cross zone connectivity
+- manage policies that are pushed to all zones
+
+## Prerequisites
+- Completed [quickstart](/docs/{{ page.version }}/quickstart/kubernetes-demo/) to setup a zone control plane with demo application
+
+## Start a global control plane
+
+### Start a new Kubernetes cluster
+
+Currently, it's not possible to deploy global and zone control plane in the same Kubernetes cluster, therefore we need a new Kubernetes cluster.
+
+{% tip %}
+You can skip this step if you already have a Kubernetes cluster running.
+It can be a cluster running locally or in a public cloud like AWS EKS, GCP GKE, etc.
+{% endtip %}
+
+```sh
+kind create cluster --name=mesh-global
+```
+
+By default, Kind does not support LoadBalancer therefore we need to configure it by following this [guide](https://kind.sigs.k8s.io/docs/user/loadbalancer/).
+Note that all public cloud providers support load balancers out of the box.
+
+### Deploy a global control plane
+
+```sh
+helm install --create-namespace --namespace {{site.mesh_namespace}} \
```

Comment:
can't use helm because it's not released, used 

```
curl -L https://kuma.io/installer.sh | VERSION=2.6.0-preview.vc1abbedd3 sh -
```

Reply:
Right, I use `kumactl install` as well to verify